specification/botservice/resource-manager/Microsoft.BotService/stable/2021-03-01/botservice.json

```diff
@@ -0,0 +1,2640 @@
+        }
+      },
+      "patch": {
+        "tags": [
+          "Channel"
+        ],
+        "description": "Updates a Channel registration for a Bot Service",
+        "operationId": "Channels_Update",
+        "x-ms-examples": {
+          "Update Bot": {
+            "$ref": "./examples/UpdateChannel.json"
+          },
+          "Update Alexa": {
+            "$ref": "./examples/UpdateAlexaChannel.json"
+          },
+          "Update Line": {
+            "$ref": "./examples/UpdateLineChannel.json"
+          },
+          "Update DirectLine Speech": {
+            "$ref": "./examples/UpdateDirectLineSpeechChannel.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/channelNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "x-ms-client-flatten": true,
+            "schema": {
+              "$ref": "#/definitions/BotChannel"
+            },
+            "description": "The parameters to provide for the created bot."
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The resource provider should return 200 (OK) to indicate that the operation completed successfully. ",
+            "schema": {
+              "$ref": "#/definitions/BotChannel"
+            }
+          },
+          "201": {
+            "description": "If resource is updated successfully, the service should return 201 (Created). Execution to continue asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/BotChannel"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed. If the resource group *or* resource does not exist, 404 (NotFound) should be returned.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Channel"
+        ],
+        "description": "Deletes a Channel registration from a Bot Service",
+        "operationId": "Channels_Delete",
+        "x-ms-examples": {
+          "Delete Bot": {
+            "$ref": "./examples/DeleteChannel.json"
+          },
+          "Delete DirectLine Speech Bot": {
+            "$ref": "./examples/DeleteDirectLineSpeechChannel.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          },
+          {
```

Comment:
why is this not using the same channelName parameter as the other operations?

Reply:
not changed from previous version of swagger